awscli/customizations/argrename.py

```diff
@@ -101,6 +101,11 @@
     'codepipeline.get-action-type.version': 'action-version',
     'ecs.*.no-enable-execute-command': 'disable-execute-command',
     'ecs.execute-command.no-interactive': 'non-interactive',
+    'mgn.*.replication-servers-security-groups-i-ds':
```

Comment:
It is a bit concerning that there are multiple ID-related parameters that have that casing as in the future it is likely the casing will stay the same for any new resource for `mgn`. I'm wondering if there is a more general approach so we do not have to continually update this casing of IDs again. Specifically, we addressed these types of acronyms way back but it looks it only handles acronyms of length 3+ and not 2+: https://github.com/boto/botocore/pull/703. I'm wondering if we can just update this heuristic and then add hidden aliases for these arguments for backwards compatibility?